packages/core/src/products/listing/redux/actions/doGetListingFacets.js

```diff
@@ -0,0 +1,27 @@
+import * as actionTypes from '../actionTypes';
+import { normalize } from 'normalizr';
+import facetsSchema from '../../../../entities/schemas/facet';
+
+export default getListingFacets => (query, config) => async dispatch => {
+  dispatch({
+    type: actionTypes.GET_LISTING_FACETS_REQUEST,
+  });
+
+  try {
+    const result = await getListingFacets(query, config);
+
+    dispatch({
+      type: actionTypes.GET_LISTING_FACETS_SUCCESS,
+      payload: normalize(result, [{ values: [[facetsSchema]] }]),
+    });
```

Comment:
Here we should return the `result` after the dispatch, in case someone wants to use the raw data given from the endpoint.

Reply:
Fixed!